samples/04_gis_analysts_data_scientists/identifying-county-names-from-incomplete-house-addresses.ipynb

```diff
@@ -0,0 +1,1903 @@
+{
```

Comment:
<p>This ("In this notebook we will build a classifier using <code>TextClassifier</code> class of <code>arcgis.learn.text</code> module.") needs to be in past tense.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
fixed